server/the-usher-openapi-spec.yaml

```diff
@@ -758,6 +799,33 @@ components:
         - client_id: client-app2
           clientname: Client Application 2
 
+# PERSONAS
```

Comment:
@k1ch  I would delete these two comment lines. I know they exist a lot in the file already but I think we should stop using them as they mess up code-folding in editors like VS Code since the indentation isn't correct.